src/components/Launch/LaunchForm/LaunchInterruptibleInput.tsx

```diff
@@ -0,0 +1,99 @@
+import { Typography } from '@material-ui/core';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import * as React from 'react';
+import { Protobuf } from 'flyteidl';
+import { useStyles } from './styles';
+import { LaunchInterruptibleInputRef } from './types';
+import { formStrings } from './constants';
+
+const isValueValid = (value: any) => {
+  return value !== undefined && value !== null;
+};
+
+export interface LaunchInterruptibleInputProps {
```

Comment:
nit: no need for `export` keyword in this inteface, as it used only in this file